lib/matplotlib/axis.py

```diff
@@ -1146,7 +1146,9 @@ def get_tightbbox(self, renderer):
 
         for a in [self.label, self.offsetText]:
             if a.get_visible():
-                bb.append(a.get_window_extent(renderer))
+                x, y = a.get_position()
+                if np.isfinite(x) and np.isfinite(y):
+                    bb.append(a.get_window_extent(renderer))
```

Comment:
Spent ~2s thinking about this but in general wouldn't it be better to first get the bbox and then include it only if it is finite?  Or does the exception occur in get_window_extent? (in which case I'd argue it should be fixed there)

In fact perhaps get_window_extend *should* return a null bbox (do we even have a way to represent that) in that case, to avoid having to special-case it again and again.  Or perhaps not (again, just throwing in some comments).

Reply:
Good point.  

However, I don't think the API allows returning a `None` for the window extent, so we'll have to check the dims of the bbox are reasonable w/o an API change